app/models/token.rb

```diff
@@ -1,6 +1,6 @@
 class Token < ActiveRecord::Base
   validates_lengths_from_database
-  belongs_to_host :foreign_key => :host_id
+  belongs_to_host_managed :foreign_key => :host_id
```

Comment:
not directly related but seems to me that `:foreign_key => :host_id` is superfluous here?